docs/source/markdown/containers-mounts.conf.5.md

```diff
@@ -10,7 +10,7 @@ The mounts.conf file specifies volume mount directories that are automatically m
 The format of the mounts.conf is the volume format `/SRC:/DEST`, one mount per line. For example, a mounts.conf with the line `/usr/share/secrets:/run/secrets` would cause the contents of the `/usr/share/secrets` directory on the host to be mounted on the `/run/secrets` directory inside the container. Setting mountpoints allows containers to use the files of the host, for instance, to use the host's subscription to some enterprise Linux distribution.
 
 ## FILES
-Some distributions may provide a `/usr/share/containers/mounts.conf` file to provide default mounts, but users can create a `/etc/containers/mounts.conf`, to specify their own special volumes to mount in the container.
+Some distributions may provide a `/usr/share/containers/mounts.conf` file to provide default mounts, but users can create a `/etc/containers/mounts.conf`, to specify their own special volumes to mount in the container. When Podman runs in rootless mode, the file `$HOME/.config/containers/mounts.conf` will override the default if it exists.
```

Comment:
`$HOME` and `$XDG_HOME_DIR` should be equivalent, but we do use $XDG_HOME_DIR and not $HOME in the code.

```suggestion
Some distributions may provide a `/usr/share/containers/mounts.conf` file to provide default mounts, but users can create a `/etc/containers/mounts.conf`, to specify their own special volumes to mount in the container. When Podman runs in rootless mode, the file `$XDG_CONFIG_HOME/.config/containers/mounts.conf` will override the default if it exists.
```

Reply:
I think `$HOME` is more consistent. While XDG_CONFIG_HOME has precedence over HOME (for other files as well), no other man page references it.

Reply:
Yep, no other man page references it, but we talk about it in some tutorials and blogs.  I brought it up just in case they were ever to resolve into different settings, currently, they are pointing at the same.  So I'm OK with this for now, but something we might need to change later.